RetrospectiveExtension.Frontend/webpack.config.js

```diff
@@ -10,6 +11,7 @@ module.exports = {
   devtool: 'source-map',
   entry: `${APP_DIR}/index.tsx`,
   output: {
+    publicPath: "/dist/",
```

Comment:
We may only want to add this in our local environment to enable break point debugger. Not sure if this does anything to our production build.